reports/src/main/java/io/jmix/reports/ReportSecurityManager.java

```diff
@@ -90,6 +90,6 @@ public void applyPoliciesByEntityParameters(LoadContext lc, @Nullable MetaClass
     }
 
     protected String wrapCodeParameterForSearch(String value) {
-        return "%," + QueryUtils.escapeForLike(value) + ",%";
+        return "%" + QueryUtils.escapeForLike(value) + "%";
```

Comment:
idk why here was %,:parameter,% and that wont work if "paramOne"
"param1,paramThatRequired,param2...." will be ok

Reply:
Commas are required to escape the role code. 

Example:
Value **roleIdx**: `system-role,client-role`

If you remove the commas, then if the user has a `system` role, the query will display a report.